### PR TITLE
ISSUE-72: Filter versioning

### PR DESCRIPTION
**Pull request by quantranhong1999**
Solving @https://github.com/linagora/openpaas-james/issues/72
need this to be merged first: @https://github.com/apache/james-project/pull/341

### CODE REVIEW

File: openpaas-james/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/openpaas/james/common/LinagoraFilterSetMethodContract.scala
Comment (chibenwa) [inline openpaas-james/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/openpaas/james/common/LinagoraFilterSetMethodContract.scala:101]: Identation

**Review by chibenwa — APPROVED** [review]

**Comment by chibenwa** [comment]
You need a rebase ?

**Review by LanKhuat — APPROVED** [review]

**Comment by chibenwa** [comment]
You still need a rebase :-p

**Comment by quantranhong1999** [comment]
> You still need a rebase :-p

Sorry my bad taking errors when rebasing both projects
Now rebased succeed

**Comment by chibenwa** [comment]
`[INFO] jmap-extensions 1.0.0-SNAPSHOT ..................... FAILURE [ 17.330 s]`

Not compiling yet, sorry...

**Comment by quantranhong1999** [comment]
> `[INFO] jmap-extensions 1.0.0-SNAPSHOT ..................... FAILURE [ 17.330 s]`
> 
> Not compiling yet, sorry...

Should I put the changes of james-project into this PR as a commit? Or should I wait until the james-project PR is merged?

**Comment by chibenwa** [comment]
Let's wait then.

**Review by Arsnael — APPROVED** [review]

**Comment by quantranhong1999** [comment]
Can someone rerun the CI for this PR?

**Comment by Arsnael** [comment]
Not updated yet, but thanks for the reminder, I will push a PR for it now!

**Pull request merged**
